conf/software-vendors.json

```diff
@@ -182,6 +182,24 @@
       "incomeAndDeductions": [
         "self-employment"
       ]
+    },
+    {
+      "name": "IRIS Accountancy Suite",
+      "url": "https://www.iris.co.uk/products/iris-accountancy-suite/",
+      "phone": "03448155555",
+      "email": "https://www.iris.co.uk/",
+      "website": "https://www.iris.co.uk/products/iris-accountancy-suite/",
+      "filters": [
+        "agent",
+        "vat",
+        "microsoft-windows",
+        "application-based",
+        "bridging"
```

Comment:
Bridging is fine, but could you confirm with BA whether or not this vendor should have record-keeping as it mentions it can be used for both in remarks

Reply:
BA is happy with only bridging since record-keeping is not mentioned in the list

Reply:
As per BA's comments record-keeping is not required

Reply:
Record-keeping is not required as per BA's comments